sdk/ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -368,55 +368,6 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
       }
     }
 
-    "query for visible contract, which was created and archived by another party before your first query" in withHttpService {
```

Comment:
This test was also moved into
`sdk/ledger-service/http-json/src/itlib/scala/http/AbstractPruningTest.scala`
which has both Postgres and Oracle concrete classes.

The difference between this version and the new one is that the new one also adds a prune step at the beginning, which is the situation we're trying to test for.